app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -11,6 +11,10 @@ class OmniauthCallbacksController < Devise::OmniauthCallbacksController
 
     def handle(provider)
       auth_hash = request.env['omniauth.auth']
+      unless auth_hash.info.email
```

Comment:
This needs to be 
`unless auth_hash.info.email.present?`

Reply:
Alright!!